dreamhome.py

```diff
@@ -1,16 +1,46 @@
-from flask import Flask
+from flask import Flask, jsonify
 from database.db import initialize_db
+from api.routes.api_routes import api
+from api.helpers.errors import APIError
+import traceback
+import pry
 
-app = Flask(__name__)
+app = Flask(__name__, template_folder = 'api/views')
 
-app.config['MONGODB_SETTINGS'] = {
-    'host': 'mongodb://localhost/dreamhome'
-}
+app.config.from_envvar('ENV_FILE_LOCATION')
```

Comment:
Okay, this code was sent to a .env file.  I thought we lost it.  This is the code that needs to be configured correctly to our heroku, because this is what links it to the Atlas MongoDB

Reply:
We need to update the runner: `dreamhome.py`? Or the `.env` files?